templates/cloud/storage.html

```diff
@@ -104,7 +104,7 @@ <h3 class="p-matrix__title">SwiftStack</h3>
                 <p>Legacy storage solutions are often priced per&dash;node or according to total disk capacity. This can lead to a penalty for good practices such as high durability&nbsp;replication.</p>
                 <p>Ubuntu Advantage Storage is different. Our pricing is based on the amount of data you record in your storage pool, regardless of how it is written or how much disk space is used.</p>
                 <p>Ubuntu Advantage Storage means you don&rsquo;t have to pay for replicas or empty space.</p>
-                <p><a href="/cloud/contact-us?product=storage">Find out how much you can save&nbsp;&rsaquo;</a></p>
+                <p><a href="/cloud/contact-us?product=cloud-storage">Find out how much you can save&nbsp;&rsaquo;</a></p>
```

Comment:
Same here. Either update the contact-us if statement or change this to just storage.